lib/builderator/patch/thor-actions.rb

```diff
@@ -40,6 +40,39 @@ def run_with_input(command, input, config = {})
     end
 
     ##
+    # Run an external command without bundler's injected environment variables
+    # (e.g. keep vagrant happy in it's own little vendor full of unicorns)
+    ##
+    def run_without_bundler(command, config = {})
+      env = ENV.clone
+      env.delete('RUBYOPT')
+      env.delete('RUBYLIB')
```

Comment:
Why not just use [Bundler.with_clean_env](http://www.rubydoc.info/github/bundler/bundler/Bundler.with_clean_env) instead?  This only removes Bundler's customizations instead of completely unsetting the var and seems more future proof.